pkg/sippyserver/server.go

```diff
@@ -1179,7 +1179,7 @@ func (s *Server) jsonJobRunIntervals(w http.ResponseWriter, req *http.Request) {
 	if len(jobName) > 0 {
 		if len(repoInfo) > 0 {
 			// GCS bucket path for presubmits
-			gcsPath = fmt.Sprintf("pr-logs/pull/%s/%s/%s/%s", repoInfo, pullNumber, jobName, jobRunIDStr)
+			gcsPath = fmt.Sprintf("pr-logs/pull/%s/%s/%s", pullNumber, jobName, jobRunIDStr)
```

Comment:
Good find.

Do non-presubmits still work? I'm confused how they possibly could with this change?

Reply:
This change affects only the situation where there's a `repoInfo` which is made up of org/repo.  The case for presubmits is unchanged (I've tried it perhaps an hour ago and know it still works).

I added the two cases in the summary for local mode.